logprep/util/helper.py

```diff
@@ -104,6 +104,13 @@ def add_field_to(event, output_field, content, extends_lists=False, overwrite_ou
     return False
 
 
+def _get_item(items, item):
+    try:
+        return dict.__getitem__(items, item)
+    except TypeError:
+        return list.__getitem__(items, int(item))
```

Comment:
looks good to me. I am not sure if there is a need for it, but it is fairly simple to extend this to get characters from a string with `str.__getitem__(items, int(item))`. As one single character might be uninteresting though you could also extend it with the slicing functionality. For that you "just" have to check if the `item` string has a `:` colon in it, split the string and pass it into the `slice(start, stop, step)` method.